.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+#### Motivation
+
+_What does this change aim to achieve?_
+
+#### Modification
+
+_Why is this change being made? What implications or other considerations are there?_
```

Comment:
Shouldn't the paragraphs be flipped?

```suggestion
#### Motivation

_Why is this change being made? What implications or other considerations are there?_

#### Modification

_What does this change aim to achieve?_
```

Reply:
This is the exact same as all the other templates https://github.com/search?q=%22_Why+is+this+change+being+made%3F+What+implications+or+other+considerations+are+there%3F_%22&type=code

Reply:
Yes, I thought this would be a good place to bring up that they all seem to have the paragraphs flipped :)